_includes/swarm-mode.md

```diff
@@ -0,0 +1,11 @@
+> **Note**
+>
+> Swarm mode is an advanced feature of Docker Engine and Docker Desktop.
+> It's useful for managing a cluster of Docker daemons.
+>
+> Swarm mode is useful if you intend to use Swarm as a production runtime
+> environment. If you're not planning on using Swarm in production,
+> use [Docker Compose](/compose/) instead.
```

Comment:
```suggestion
> environment. If you're not planning on deploying with Swarm,
> you should likely prefer [Docker Compose](/compose/) instead.
```

Reply:
I'll change the first part to say "deploying with Swarm," but I don't think "you should likely prefer" is better wording than "use".

Reply:
Fair enough; I was trying to convey "this makes sense for most users," but honestly "use" is probably fine -- if this advice doesn't apply to you, you likely know.